### ISSUE TITLE
not work when :set mouse=a

### ISSUE THREAD

**Issue by futurist** [terryma/vim-multiple-cursors#194]
when `:set mouse=a`
It's not worked tested in vim 7.4

**Comment by faceleg** [comment]
Please read https://github.com/terryma/vim-multiple-cursors/blob/master/CONTRIBUTING.md and updated your issue

**Comment by christianrondeau** [comment]
Since I have the same issue:

# Problems summary

When launching VIM with vim-multiple-cursors and `set mouse=a` in my `vimrc`, I notice multiple issues:

* When I launch VIM, the command window shows `:.4205;0c`
* When I try to click in an empty buffer, I receive `E348: No string under cursor`
* When I click on text in a non-empty buffer, I receive various errors, such as `E486: Pattern not found: \v \\` and `E78: Unknown mark` and sometimes the command line shows `!?`, `Invalid Range`, and broken regex searches (as far as I can tell, it seems random)
* Using the mouse wheel switch to other buffers, inserts some text and otherwise behaves erratically.

# Expected

* I can click to move the cursor normally
* I can use the mouse whell to scroll the buffer
* VIM starts without pre-entered characters in command line

# Environment Information

OS: Tried on Ubuntu 16 and Termux
Neovim/Vim/Gvim version: 7.4 and 8.0 (Issue ONLY in VIM, NOT in GVIM)
Provide a minimal .vimrc with less than 50 lines:

```
" Given you have a valid vim-plug setup:
set mouse=a
Plug 'terryma/vim-multiple-cursors'
```

# In short

Simply add set mouse=a, launch VIM (not GVIM) and you should see the issues. I'm not sure where to start.

# Workarounds

Only enable the mouse in normal mode (`set mouse=n`) seems to do the trick.

**Comment by jcdescy** [comment]
I can not replicate this with Vim 8.0.1300 on Mac OS X Sierra in iTerm2. All seems well here.

**Comment by antoinemadec** [comment]
Unable to reproduce on Vim 8.0 Ubuntu 16.04.
My vimrc also features ```set mouse=a``` and I never had any issue.
Closing issue.

**Comment by christianrondeau** [comment]
I just checked (I had vim-multiple-cursors disabled whenever I was not in gvim since 2016) and it works :)

I verified on VIM 7.4 with patch 1689 on Min 18.1, and Vim 8.0 with patch 1600 on Termux, both worked flawlessly with `mouse=a`. I guess this might have been fixed by "something" since then...
